TradeItIosEmsApi/TradeItOAuthAccessTokenResult.h

```diff
@@ -3,5 +3,6 @@
 @interface TradeItOAuthAccessTokenResult: TradeItAuthLinkResult
 
 @property (nullable) NSString *broker;
+@property (nullable) NSString *activationTime; // IMMEDIATE if there is no delay using the linked account, otherwise: ONE_OR_TWO_BUSINESS_DAY
```

Comment:
What happens if another broker comes along that is 1 week? Is that likely to happen? @TheAntonioReyes